### PR TITLE
starting on the readme

### PR DESCRIPTION
**Pull request by asharnaud**
Hey @pablo-jurado what do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Restaurant Group Project
```
Comment (pablo-jurado) [inline README.md:1]: Nice job using h1 tag 👍

**Comment by pablo-jurado** [comment]
This is a nice start but we need more work. Go team!

**Comment by asharnaud** [comment]
I think it looks better now @pablo-jurado

**Comment by pablo-jurado** [comment]
LGTM 👍

**Pull request merged**